### PR TITLE
Move to shine-oss Azure DevOps project

### PR DESCRIPTION
**Pull request by kmahone**
For validation pipelines we are moving from the 'ms' Azure DevOps project to 'shine-oss'.

This updates the url for the dependencies nuget feed.

This also includes some fixes to get the validation pipelines passing again. There were some updates to the C++ compiler that caused us to hit new build errors.

### CODE REVIEW

File: build/MUX-PR.yml [hunk 1]
```diff
@@ -25,11 +25,12 @@ stages:
...
     pool: 
-      ${{ if eq(variables['System.CollectionUri'], 'https://dev.azure.com/ms/') }}:
+      ${{ if eq(variables['System.CollectionUri'], 'https://dev.azure.com/shine-oss/') }}:
         name: WinDevPoolOSS-L
-      ${{ if ne(variables['System.CollectionUri'], 'https://dev.azure.com/ms/') }}:
+      ${{ if ne(variables['System.CollectionUri'], 'https://dev.azure.com/shine-oss/') }}:
         name: WinDevPool-L
-      demands: ImageOverride -equals WinDevVS17-latest
+      demands: 
+        - ImageOverride -equals WinDevVS17-latest
```
Comment (RBrid) [inline build/MUX-PR.yml:33]: Did you really intend to make this change?   It's not consistent with the build/MUX-CI.yml file for instance.

Comment (kmahone) [reply build/MUX-PR.yml:33]: it's just an alternative syntax. This syntax lets you add multiple demands (one per line). I was using the version override demand during testing.

**Review by RBrid — APPROVED** [review]

**Pull request merged**